src/public-lib/components/checkedSelect/Checklist.tsx

```diff
@@ -0,0 +1,89 @@
+import autobind from "autobind-decorator";
+import * as _ from "lodash";
+import {action, computed} from "mobx";
+import {observer} from "mobx-react";
+import * as React from "react";
+
+import {getOptionLabel, getSelectedValuesMap, Option} from "./CheckedSelectUtils";
+
+type ChecklistProps = {
+    onChange: (values: {value: string}[]) => void;
+    value: {value: string}[];
+    options: Option[];
+    isDisabled?: boolean;
+    numberOfColumnsPerRow?: number;
+};
+
+function generateTableRows(selectComponents: JSX.Element[], numberOfColumnsPerRow: number = 1)
+{
+    return _.chunk(selectComponents, numberOfColumnsPerRow)
+        .map(components => <tr>{components}</tr>);
+}
+
+@observer
+export default class Checklist extends React.Component<ChecklistProps, {}>
+{
+    public static defaultProps: Partial<ChecklistProps> = {
+        isDisabled: false,
+        numberOfColumnsPerRow: 1
+    };
+
+    @computed
+    get selectedValues() {
+        return getSelectedValuesMap(this.props.value);
+    }
+
+    public render() {
+        return (
+            <table>
```

Comment:
is there a reason to use table to lay this out instead of bootstrap form layout?

Reply:
We've been using the same structure for the original component in mutations tab, I just followed that example: https://github.com/cBioPortal/cbioportal-frontend/blob/c1388e15d02f7b4d46e5fa338100cccbb83d4250/src/shared/components/mutationTypePanel/ProteinImpactTypePanel.tsx#L44

Is it easy with bootstrap to force a `2x2` layout without line wrapping?